### DISCUSSION TITLE
Background dim crashed game

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#22231] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

Changed my background dim to 100%

### Screenshots or videos

https://user-images.githubusercontent.com/40531871/212624934-2403c6b5-ac80-46bd-a419-d4ff451eb535.mp4

### Version

2022.1228.0

### Logs

[network.log](https://github.com/ppy/osu/files/10423561/network.log)
[performance.log](https://github.com/ppy/osu/files/10423562/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/10423563/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/10423564/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/10423566/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/10423567/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/10423568/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10423569/updater.log)
[database.log](https://github.com/ppy/osu/files/10423570/database.log)
[input.log](https://github.com/ppy/osu/files/10423571/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/10423572/legacy-ipc.log)

**Comment by peppy** [comment, marked as answer]
Logs don't show any sign of the game crashing itself. Please check windows event viewer for any native crash details.

**Reply by AuracleTech** [reply]
First error `Application Error`
Faulting application name: osu!.exe, version: 0.0.0.0, time stamp: 0x61e1d8d9
Faulting module name: coreclr.dll, version: 6.0.222.6406, time stamp: 0x61e1d09e
Exception code: 0xc0000005
Fault offset: 0x00000000001cd72f
Faulting process id: 0x1834
Faulting application start time: 0x01d9297a696c2299
Faulting application path: C:\Users\Admin\AppData\Local\osulazer\app-2022.1228.0\osu!.exe
Faulting module path: C:\Users\Admin\AppData\Local\osulazer\app-2022.1228.0\coreclr.dll
Report Id: eecc3043-3184-4a9c-9dc4-530e03ab2033
Faulting package full name: 
Faulting package-relative application ID: 
Second error `.NET Runtime`
Application: osu!.exe
CoreCLR Version: 6.0.222.6406
.NET Version: 6.0.2
Description: The process was terminated due to an internal error in the .NET Runtime at IP 00007FFBE865D72F (00007FFBE8490000) with exit code c0000005.
![image](https://user-images.githubusercontent.com/40531871/212631900-169710ad-35d7-45e6-a890-eaeaf4d5e5b4.png)

Appears to be an `Access violation error` which is probably osu-framework

**Reply by peppy** [reply]
That's still quite weird because it should be logged. Are you able to reproduce it reliably?

**Reply by bdach** [reply]
It's a bog-standard access violation, have seen this many times over. I'm gonna claim driver error.

Would try updating drivers, disabling shadowplay if enabled, disabling any other overlays. If all of that fails probably best wait for new renderer.

**Reply by AuracleTech** [reply]
> Are you able to reproduce it reliably?

Can't reproduce :/ guess the dimming was probably non related
I am using `outplayed` which uses `overwolf`'s API to register my replays
Could it be possible `overwolf`' tries to read osu stable data from osu lazer and cause the crash?

**Reply by bdach** [reply]
Possible. Anything that is going to interfere with the game in some respect may trigger this too.
